filter/filter_test.go

```diff
@@ -150,7 +170,7 @@ func TestFilters(t *testing.T) {
 	tzng = `
 #0:record[nested:record[field:string]]
 0:[[test;]]`
-	runCases(t, tzng, []testcase{
+	runCasesExpectBufferFilterFalsePositives(t, tzng, []testcase{
```

Comment:
Would you please add a comment explaining why these trigger bf false positives?

Reply:
Done in 003f682b0b6e625ca19292606a27fc7452b6651b.